engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -89,6 +89,12 @@
       for an element that has a flow scope with more than one active instance: '%s'. Can't decide \
       in which instance of the flow scope the element should be activated.""";
 
+  private static final String ERROR_MESSAGE_CHILD_PROCESS_INSTANCE_TERMINATED =
+      """
+      Expected to modify instance of process '%s' but the given instructions would terminate \
+      the instance. A different process started this process. To terminate this instance \
+      please modify the parent process instead.""";
```

Comment:
:wrench: We could be a bit more concrete in the rejection message by mentioning the call activity.

```suggestion
      Expected to modify instance of process '%s' but the given instructions would terminate \
      the instance. The instance was created by a call activity in the parent process instance. \
      To terminate this child process instance, modify the call activity in the parent process \
      instance instead.""";
```